backend/api/Areas/Property/Models/Parcel/BuildingParcelModel.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Pims.Api.Areas.Property.Models.Parcel
+{
+    public class BuildingParcelModel : ParcelModel
```

Comment:
we want all the fields in the parcel model, plus a few more since this parcel data is specific to the building -> parcel relationship.

Reply:
I don't fetch the Parcels -> Buildings relationship right now, so it ends up being Building -> Parcels -> []